components/authz-service/server/v2/system_integration_test.go

```diff
@@ -160,7 +160,7 @@ func TestListProjectRules(t *testing.T) {
 
 		expectedRules1 := []*api.ProjectRule{createResp1.Rule, createResp2.Rule}
 		expectedRules2 := []*api.ProjectRule{createResp3.Rule}
-
+		
```

Comment:
is this spaces? looks funny

Reply:
It's superfluous at any rate ;) `gofmt -w` should remove these. 😃